docs/overview/patterns-and-practices-overview.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Overview of Patterns and Practices for Office Add-ins
```

Comment:
```suggestion
title: Overview of Patterns and Practices (PnP) for Office Add-ins
```
I think this will improve SEO, search terms are high for "PnP"